tsconfig.test.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "module": "es6",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "noImplicitAny": false,
```

Comment:
Added this because it was really annoying fighting with it in unit tests.

Reply:
That's what this is doing because it's in `tsconfig.test.json` unless you're referring to something else.